leet_69.py

```diff
@@ -0,0 +1,7 @@
+import math
+class Solution:
+    def mySqrt() -> int:
+        x = int(input("Please enter an integer:"))
+        rt = int(math.sqrt(x))
+        print(rt)
+    mySqrt()
```

Comment:
No need to call function in this way, leetcode will automatically call `Solution.mySqrt` and pass input x automatically.

Reply:
Thank you for all the feedback! I will open a new pull request with the correct file.